File: Foundation/Patterns/triangle.cpp
#include<iostream>
using namespace std;

void printTriangle(int row)
{
    int nst = 1;
    for(int r = 1; r <= row; r++)
    {
        for(int cst = 1; cst <= nst; cst++)
        {
            cout<<"*\t";
        }
        nst++;
        cout<<endl;
    }
}

int main()
{
    int row;
    cin>>row;
    printTriangle(row);
    return 0;
}

File: Foundation/Patterns/triangle.java
import java.util.Scanner;
public class triangle
{
    public static Scanner scn = new Scanner(System.in);

    public static void triangle(int row)
    {
        int nst = 1;
        for(int r = 1; r <= row; r++)
        {
            for(int cst = 1; cst <= nst; cst++)
            {
                System.out.print("*\t");
            }
            nst++;
            System.out.println();
        }
    }

    public static void main(String[] args)
    {
        int n = scn.nextInt();
        triangle(n);
    }
}